src/docker.rs

```diff
@@ -1317,3 +1426,6 @@ pub(crate) type BodyType = http_body_util::Either<
 pub(crate) fn body_stream(body: impl Stream<Item = Bytes> + Send + 'static) -> BodyType {
     BodyType::Right(StreamBody::new(Box::pin(body.map(|a| Ok(Frame::data(a))))))
 }
+
+/// A request from bollard used in callbacks
+pub type BollardRequest = Request<BodyType>;
```

Comment:
I'm happy where this is going... I think we can merge soon.

Just a nitpic - could we remove the alias and use the full type where it is used (e.g. the documentation). This makes it clearer what it is at the expense of verbosity.

Reply:
This probably mean that `BodyType` must become fully public